### DISCUSSION TITLE
Aeon labs ZW095 missing association groups 1-3

### DISCUSSION THREAD

**Discussion by TheKorn2** [zwave-js/zwave-js#6683] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

**THIS IS NOT A DUPLICATE.**  Same bug affects second device type!

Aeon Labs ZW095 has a brazillion configuration options on reports to send to association groups 1-3. However, groups 1-3 are not defined, so you can't add any nodes to be sent the reports.

1.  Add a ZW095
2.  Click on groups
3.  See lifeline group
4.  Hit add
5.  no groups but lifeline available.

### Software versions

zwave-js-ui: 9.8.2.c09b638
zwave-js: 12.4.3

Not using home assistant, but am using zwave-js-ui.

### Device information

Manufacturer: Aeon Labs
Model name: ZW095
Node ID: 123

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-02-20.zip](https://github.com/zwave-js/node-zwave-js/files/14353501/zwavejs_2024-02-20.zip)

**Comment by AlCalzone** [comment, marked as answer]
The answer is the same though - association groups 2 and 3 do not exist. The config parameter refer to report groups you define entirely through the configuration parameters. See also https://github.com/zwave-js/node-zwave-js/discussions/6682#discussioncomment-8541954
